src/pages/MoviesPage.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="watchable-list-page">
+    <div class="feed-title">{{ title }}</div>
+    <div class="watchable-blocks row q-col-gutter-lg">
+      <WatchableBlock
```

Comment:
Kodel WatchablesFeedPage padarei i du skirtingus page? Kartojasi kodas labai, prisikabint gali kad nesilaikome DRY. WatchablesFeedPage buvo ideja tokia, kad paiemi propus priklausomai nuo to ar yra SeriesPage ar MoviesPage ir atitinkamus parodai fieldus/iškvieti metodus iš bendro feed-store. Aišku jei norisi seperation of concerns labai, galima turėti tuos storus atskirus, kaip ir dabar padaręs esi series-store ir movie-store, bet kviesti iš WatchableListPage skirtingus store pagal gautą prop ar tai series ar movies list page.